### PR TITLE
Feat: Add MyCollectionInfo to Me

### PR DESCRIPTION
**Pull request by lordkiz**
Addresses [CX-1868]

Allows us to get general info about the current user's MyCollection. This is needed to extract info such as whether the user's myCollection includesPurchasedArtworks.

[CX-1868]: https://artsyproduct.atlassian.net/browse/CX-1868?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/schema/v2/me/myCollection.ts [hunk 1]
```diff
@@ -31,6 +31,10 @@ const MyCollectionConnection = connectionWithCursorInfo({
     default: {
       type: new GraphQLNonNull(GraphQLBoolean),
     },
+    includesPurchasedArtworks: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      resolve: (it) => it.includes_purchased_artworks,
```
Comment (MounirDhahri) [inline src/schema/v2/me/myCollection.ts:36]:
```suggestion
      resolve: (myCollection) => myCollection.includes_purchased_artworks,
```

File: _schemaV2.graphql [hunk 2]
```diff
@@ -8436,7 +8436,7 @@ type Me implements Node {
     before: String
 
     # Exclude artworks that have been purchased on Artsy and automatically added to the collection.
-    excludePurchasedArtworks: Boolean
+    excludePurchasedArtworks: Boolean = false
```
Comment (MounirDhahri) [inline _schemaV2.graphql:8439]: **issue:** do you know why this occurred although you didn't change it in the schema?

**Review by MounirDhahri — COMMENTED** [review]
Nice. I left 2 small comments and I believe you forgot to update the tests.

File: _schemaV2.graphql (hunk 2 above)
Comment (lordkiz) [reply _schemaV2.graphql:8439]: This is coming from this: https://github.com/artsy/metaphysics/blob/faa9ec1113cd2133b5d2d65d1895845682b83dfd/src/schema/v2/me/myCollection.ts#L72

Comment (MounirDhahri) [reply _schemaV2.graphql:8439]: weird, this should already be in the schema since it's not code that you added. not sure why it only got added now.

**Review by MounirDhahri — APPROVED** [review]
LGTM 👏 would you mind if you merge master then feel free to merge

**Review by rajsam003 — APPROVED** [review]
LGTM

**Pull request merged**